imas_paraview/tests/test_beam.py

```diff
@@ -1,22 +1,16 @@
-import pytest
 from imas import DBEntry
 from vtkmodules.vtkCommonDataModel import vtkMultiBlockDataSet
 
 from imas_paraview.plugins.beam import BeamReader
 
 
-@pytest.mark.skip(reason="no IMAS-Core available")
 def test_load_beam():
     """Test if limiters are loaded in the VTK Multiblock Dataset."""
     reader = BeamReader()
     with DBEntry(
-        (
-            "imas:hdf5?path=/work/imas/shared/imasdb/ITER_MACHINE_DESCRIPTION/"
-            "3/120000/1304/"
-        ),
-        "r",
+        "/home/ITER/blokhus/public/imas_paraview_tests/iter_md-120000-1304.nc", "r"
```

Comment:
Perhaps you could make the "base folder" where the test data is stored a fixture (in conftest.py).

Once the datasets are published, we would only need to update one fixture instead of every test file.